Rethinking_2/Chp_08.ipynb

```diff
@@ -0,0 +1,1578 @@
+{
```

Comment:
<p>this should be </p><p> </p><p>np.sum(np.abs(m_8_1_prior["b"]) &gt; 0.6) / len(m_8_1_prior["b"])</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/88/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>